.buildkite/pipeline.yml

```diff
@@ -32,9 +32,6 @@ steps:
         echo "--- :npm: Install Node dependencies"
         npm ci --unsafe-perm --prefer-offline --no-audit --no-progress
 
-        echo "--- :package: Run bundle prep work"
-        npm run prebundle:js
-
```

Comment:
I skipped this step to speed up the bundle generation, it should be reverted before merging the PR.

Reply:
I'm surprised the build worked without it 🤔 Maybe it's no longer necessary and we should not add it back?

Reply:
The `prebundle` is needed to include the localizations, but skipping it doesn't break the editor. If it's not run, the editor would be only in English.

Reply:
Thank you for the explanation @fluiddot.

> In the end, we'd only need the optimal bundle when releasing a new version of Gutenberg Mobile, so we might add a condition to the job to run `prebundle` in that situation.

I see... We could do this by checking if the CI build is running on a tag